cpp/src/converters/COOtoCSR.cu

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
encoding change?

Reply:
Not sure what your question means.  I added direct python bindings to, so I needed a .cu file to compile the correct templated instances.

Reply:
got it